### PR TITLE
Demos: Fix datepicker widget by including ui-core

### LINKED ISSUE #8183: Datepicker: Popup demo broken on 1.5-dev
The "Popup Datepicker" demo is broken. The widget fails to pop up when the user clicks into the textinput.

### PR DESCRIPTION
**Pull request by apsdehal**
Fixes #8183
Closes #8445

### CODE REVIEW

File: demos/datepicker/index.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <script src="../../external/jquery/jquery.js"></script>
     <script src="../_assets/js/"></script>
     <script src="../../js/"></script>
+    <script src="http://cdn.rawgit.com/jquery/jquery-ui/1.10.4/ui/jquery.ui.core.js"></script>
```
Comment (arschmitz) [inline demos/datepicker/index.php]: should update this to a newer version 1.12 is in RC

Comment (apsdehal) [inline demos/datepicker/index.php]: Working on it.

File: demos/datepicker/index.php [hunk 2]
```diff
@@ -8,13 +8,13 @@
...
+    <link rel="stylesheet" href="http://cdn.rawgit.com/arschmitz/jquery-mobile-datepicker-wrapper/v0.1.9/jquery.mobile.datepicker.css">
+    <link rel="stylesheet" href="http://cdn.rawgit.com/arschmitz/jquery-mobile-datepicker-wrapper/v0.1.9/jquery.mobile.datepicker.theme.css">
     <script src="../../external/jquery/jquery.js"></script>
     <script src="../_assets/js/"></script>
     <script src="../../js/"></script>
-    <script src="http://cdn.rawgit.com/jquery/jquery-ui/1.10.4/ui/jquery.ui.core.js"></script>
-    <script src="http://cdn.rawgit.com/jquery/jquery-ui/1.10.4/ui/jquery.ui.datepicker.js"></script>
-    <script id="mobile-datepicker" src="http://cdn.rawgit.com/arschmitz/jquery-mobile-datepicker-wrapper/v0.1.1/jquery.mobile.datepicker.js"></script>
+    <script src="http://cdn.rawgit.com/jquery/jquery-ui/1.12.0-rc.2/ui/widgets/datepicker.js"></script>
+    <script id="mobile-datepicker" src="http://cdn.rawgit.com/arschmitz/jquery-mobile-datepicker-wrapper/v0.1.9/jquery.mobile.datepicker.js"></script>
```
Comment (arschmitz) [inline demos/datepicker/index.php]: thinking maybe we should use bower to manage these files and put them in the external directory

**Comment by arschmitz** [comment]
👍

**Pull request closed without merging**